src/lambda/models/samLambdaRuntime.ts

```diff
@@ -32,7 +37,7 @@ export const pythonRuntimes: ImmutableSet<Runtime> = ImmutableSet<Runtime>([
 ])
 export const dotNetRuntimes: ImmutableSet<Runtime> = ImmutableSet<Runtime>(['dotnetcore2.1', 'dotnetcore3.1'])
 const DEFAULT_RUNTIMES = ImmutableMap<RuntimeFamily, Runtime>([
-    [RuntimeFamily.NodeJS, 'nodejs12.x'],
+    [RuntimeFamily.NodeJS, 'nodejs14.x'],
```

Comment:
A couple questions:
* Should we make this the default if we're not going to bump the SAM CLI minver?
* Should we also bump `dotnetcore2.1`?

Reply:
reverted default to nodejs12 for now.